.github/workflows/build.yaml

```diff
@@ -0,0 +1,85 @@
+name: Build
+
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - '*'
+  pull_request:
+    branches:
+    - master
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  # Increment this key to invalidate the cache.
+  CACHE_KEY: 0
```

Comment:
GitHub doesn't provide a way to invalidate caches. Later, this variable is used as part of the cache key, so incrementing it will cause a cache miss. Useful if there's a problem with the cache, which I've run into before.